pytripgui/view/qt_view_adapter.py

```diff
@@ -172,6 +174,14 @@ def checked_items(self):
                 selected.append(widget.data(Qt.UserRole))
         return selected
 
+    def tick_checkboxes(self, items, get_name: Callable):
+        items_to_be_ticked = [
+            list_item for list_item in self._items
+            if get_name(list_item.data(Qt.UserRole)) in [get_name(item) for item in items]
```

Comment:
To optimize the code I'd create a dedicated variable first:
```
item_names = [get_name(item) for item in items]
items_to_be_ticked = [
            list_item for list_item in self._items
            if get_name(list_item.data(Qt.UserRole)) in item_names
]
```

Reply:
Done